### PR TITLE
CLDR-18014 site: update generating-charts

### PR DESCRIPTION
**Pull request by srl295**
CLDR-18014

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/site/development/cldr-big-red-switch/generating-charts.md [hunk 1]
```diff
@@ -2,27 +2,26 @@
 title: Generating Charts
 ---
 
-# Generating Charts
-
 ## Generate
 
-The input for this is the cldr\-staging/production file, and the output is in github cldr\-staging. **(If for the development version, the input is main or the maint branch, but we should change that.)**
+The input for this process is the `../cldr-staging/production` file, and the output is in the github repo [`cldr-staging`](https://github.com/unicode-org/cldr-staging). **(If for the development version, the input is `main` or the `maint` branch, but we should change that.)**   `cldr-staging` is expected to be checked out as a _sibling_ directory to your CLDR source repository, hence `../cldr-staging`.
```
Comment (srl295) [inline docs/site/development/cldr-big-red-switch/generating-charts.md:7]: Does GenerateAllCharts really read from cldr-staging?

Comment (srl295) [reply docs/site/development/cldr-big-red-switch/generating-charts.md:7]: also I wonder if we need a separate page discussing production data and cldr-staging - nothing else on the site about it now

File: docs/site/development/cldr-big-red-switch/generating-charts.md [hunk 2]
```diff
@@ -37,30 +36,30 @@ The input for this is the cldr\-staging/production file, and the output is in gi
 1. VM Arguments
 	1. \-DCHART\_VERSION\=**99**
 	2. \-DCHART\_STATUS\=**trunk** (uses trunk, no β. Used at the end of the release, but before the final data is in cldr\-archive)
-2. In the printout from delta\_summary.txt, there is a listing of the sizes at the top
+2. In the printout from `tsv/delta_summary.tsv`, there is a listing of the sizes at the top
 	1. Something like the following:
 		1. \# dir file added deleted changed total
 			1. TOTAL 30,276 3,601 10,909 2,153,094
-	2. Add those new figures to the release page
+	2. Add those new figures to the release page _(TODO: where?)_
```
Comment (srl295) [inline docs/site/development/cldr-big-red-switch/generating-charts.md:43]: @macchiati @AEApple I couldn't find where we were doing this in the last couple of releases.

For 46 it's

```
# dir	file	added	deleted	changed	total
TOTAL		411,778	37,224	56,077	2,719,284
```

Comment (macchiati) [reply docs/site/development/cldr-big-red-switch/generating-charts.md:43]: We have only added those when the data looks interesting, probably not for a few releases. They used to go near the top. I wouldn't spend time on this now; we could always add later to the release if we wanted.

**Review by macchiati — APPROVED** [review]
All these changes are reasonable.

**Pull request merged**